src/applications/personalization/profile360/components/BankInfoForm.jsx

```diff
@@ -58,8 +58,9 @@ const BankInfoForm = ({
   onClose,
 }) => (
   <SchemaForm
-    // name and title are used internally by the SchemaForm
+    addNameAttribute
```

Comment:
This is a relatively new feature added to `SchemaForm` as of https://github.com/department-of-veterans-affairs/vets-website/issues/13256